index.d.ts doesn't have inverted prop

## Environment

```
System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
    Memory: 2.82 GB / 15.86 GB
  Binaries:
    Node: 10.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.12.3 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
  IDEs:
    Android Studio: Version  3.3.0.0 AI-182.5107.16.33.5314842

```
## Description

TS doesn't compile if I add `inverted` prop to the slider.

It shows this error:

```
error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: Readonly<SliderProps>): Slider', gave the following error.    Type '{ value: number; onValueChange: (value: number) => void; minimumValue: number; maximumValue: number; step: number; inverted: true; minimumTrackTintColor: string; 
thumbImage: any; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Slider> & Readonly<SliderProps> & Readonly<{ children?: ReactNode; }>'.       
      Property 'inverted' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Slider> & Readonly<SliderProps> & Readonly<{ children?: ReactNode; }>'.
  Overload 2 of 2, '(props: SliderProps, context?: any): Slider', gave the following error.
    Type '{ value: number; onValueChange: (value: number) => void; minimumValue: number; maximumValue: number; step: number; inverted: 
true; minimumTrackTintColor: string; thumbImage: any; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Slider> & Readonly<SliderProps> & Readonly<{ children?: ReactNode; }>'.
      Property 'inverted' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Slider> & Readonly<SliderProps> & Readonly<{ children?: ReactNode; }>'.

126       <Slider
           ~~~~~~
```

## Reproducible Demo

I can't seem to run it, so I will just paste this code here:

```
      <Slider
        value={30}
        onValueChange={(value: number) =>{}}
        minimumValue={1}
        maximumValue={1000}
        step={1}
        inverted
        minimumTrackTintColor="blue"
      />
```

Thanks, anyone else run into this issue?
I will create a PR to fix this if it's not only me.

--- a/src/typings/index.d.ts
+++ b/src/typings/index.d.ts
@@ -102,6 +102,12 @@ export interface SliderProps extends SliderPropsIOS, SliderPropsAndroid {
    * the value during dragging.
    */
   value?: number;
+
+  /**
+   * Reverses the direction of the slider.
+   */
+  inverted?: boolean;
+
 }
 
 /**

